index.html

```diff
@@ -69,5 +69,310 @@ <h2>I'm Shakespear Glad to see you!</h2>
+                <h2>Tonic</h2>
+            </div>
+            <ul class="toniclists1">
+                <li class="canopy">
+                    Canopy
+                </li>
+                <li class="back">
+                    Back End Dev  
+                </li>
+                <li class="twenty">
+                    2015
+                </li>
+            </ul>
+            <div class="tonictext">
+                <p>
+                    A daily section of privately personalized reads; no accounts or sign-ups required
+                </p>
+            </div>
+            <ul class="skills">
+                <li>
+                    html
+                </li>
+                <li>
+                    CSS
+                </li>
+                <li>
+                    JavaScript
+                </li>
+            </ul>       
+                <button type="button" class="cardbut">
+                    See Project
+                </button>
+        </div>
+        <div class="multi2">
+            <div class="tonicimage">
+                <img src="img/Snapshoot Portfolio4.png" alt="snapshot">
+            </div>
+            <div class="tonichead1">
+                <h2>Multi-Post <br> Stories</h2>
+            </div>
+            <ul class="toniclists1">
+                <li class="canopy">
+                    Canopy
+                </li>
+                <li class="back">
+                    Back End Dev  
+                </li>
+                <li class="twenty">
+                    2015
+                </li>
+            </ul>
+            <div class="tonictext">
+                <p>
+                    A daily section of privately personalized reads; no accounts or sign-ups required
+                </p>
+            </div>
+            <ul class="skills">
+                <li >
+                    html
+                </li>
+                <li>
+                    CSS
+                </li>
+                <li>
+                    JavaScript
+                </li>
+            </ul>
+            <button type="button" class="cardbut">
+                See Project
+            </button>
+        </div>
+    </div>
+<!-- ABOUT SECTION BEGINS -->
+    <div class="lastsection">
+        <div class="about">
+            <div class="middle">
+            <div class="text1">
+                <h2>About <br> Myself</h2>
+            </div>
+            <div class="text2">
+                <p>
+                    I am an Information and Communication Engineer and full stack website developer. I can help you build a product, feature or website look throug some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me. 
+                </p>
+            </div>
+           <br>
+           <br>
+            <div class="connect">
+                <div class="text3">
+                    <h1>LET'S CONNECT </h1>
+                </div>
```

Comment:
- [ ] As the previous reviewer suggested, kindly wrap your `LET'S CONNECT` text in an anchor tag as the previous reviewer has suggested 👍🏽 as the link should take you to the contact page or another page 👍🏽 

e.g.
```
<div class="text3">
         <a href="#">LET'S CONNECT </a>
   </div>
```